### PR TITLE
regenerated StorageManagementClient with new version of the code generator

### PR DESCRIPTION
**Pull request by amarzavery**

### CODE REVIEW

File: lib/services/storageManagement2/lib/models/storageAccountUpdateParameters.js [hunk 1]
```diff
@@ -40,107 +40,59 @@ var util = require('util');
...
+                }
+            }
+          }
+        },
+        accountType: {
+          required: false,
+          serializedName: 'properties.accountType',
+          type: {
+            name: 'Enum',
+            allowedValues: [ 'Standard_LRS', 'Standard_ZRS', 'Standard_GRS', 'Standard_RAGRS', 'Premium_LRS' ]
```
Comment (yaxia) [inline lib/services/storageManagement2/lib/models/storageAccountUpdateParameters.js]: Premium_LRS and Standard_ZRS accounts cannot be changed to other account types, and other account types cannot be changed to Standard_ZRS or Premium_LRS. So these two should not in the allowed values set.

File: runtime/ms-rest/package.json [hunk 2]
```diff
@@ -5,14 +5,14 @@
     "email": "azsdkteam@microsoft.com",
     "url": "https://github.com/Azure/AutoRest"
   },
-  "version": "1.2.0",
+  "version": "1.5.0",
   "description": "Client Runtime for nodeJS client libraries generated using AutoRest",
```
Comment (yaxia) [inline runtime/ms-rest/package.json]: Minor: nodeJS may not be a common case in the community. maybe: Node.js?

File: runtime/ms-rest/test/data/TestClient/lib/models/productListResult.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+ * license information.
+ * 
+ * Code generated by Microsoft (R) AutoRest Code Generator 0.14.0.0
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+'use strict';
+
+var models = require('./index');
```
Comment (yaxia) [inline runtime/ms-rest/test/data/TestClient/lib/models/productListResult.js]: It looks "models" and "util" are not used in the generated code.

File: runtime/ms-rest/test/data/TestClient/lib/models/productListResultNextLink.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+ * Code generated by Microsoft (R) AutoRest Code Generator 0.14.0.0
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+'use strict';
+
+var models = require('./index');
+
+var util = require('util');
```
Comment (yaxia) [inline runtime/ms-rest/test/data/TestClient/lib/models/productListResultNextLink.js]: Same as above

File: runtime/ms-rest/test/data/TestClient/lib/testClient.js [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+/* jshint latedef:false */
+/* jshint forin:false */
+/* jshint noempty:false */
+
+'use strict';
+
+var util = require('util');
+var msRest = require('../../../../lib/msRest');
+var ServiceClient = msRest.ServiceClient;
+var WebResource = msRest.WebResource;
```
Comment (yaxia) [inline runtime/ms-rest/test/data/TestClient/lib/testClient.js]: It seems not be used.

**Comment by yaxia** [comment]
Except the comments I left, others look good to me.

File: lib/services/storageManagement2/lib/models/storageAccountUpdateParameters.js (hunk 1 above)
Comment (amarzavery) [inline lib/services/storageManagement2/lib/models/storageAccountUpdateParameters.js]: @yaxia - These values are coming from the [swagger doc](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2015-06-15/swagger/storage.json) maintained by your team.

File: runtime/ms-rest/package.json (hunk 2 above)
Comment (amarzavery) [inline runtime/ms-rest/package.json]: will fix the naming in the next version :).

File: lib/services/storageManagement2/lib/models/storageAccountUpdateParameters.js (hunk 1 above)
Comment (yaxia) [inline lib/services/storageManagement2/lib/models/storageAccountUpdateParameters.js]: Add @emgerner-msft to comment.

**Pull request merged**